code/CodeTester.cpp

```diff
@@ -169,6 +171,17 @@ participant_attr.rtps.useBuiltinTransports = false;
 participant_attr.rtps.userTransports.push_back(tcp_transport);
 //!--
 
+//CONF-SHM-TRANSPORT-SETTING
+//Create a descriptor for the new transport.
+auto shm_transport = std::make_shared<SharedMemTransportDescriptor>();
```

Comment:
I'd rather see the type here (haven't checked suggestion's line length)
```suggestion
std::shared_ptr<SharedMemTransportDescriptor> shm_transport = std::make_shared<SharedMemTransportDescriptor>();
```